packages/tables/resources/js/components/table.js

```diff
@@ -61,9 +61,13 @@ export default function table() {
             this.isLoading = false
         },
 
-        getRecordsInGroupOnPage: function (group) {
+        getRecordsInGroupOnPage: function (group) {            
```

Comment:
Why extra space?

Reply:
Figured code style actions would have taken care of that.

Reply:
Unfortunately we can't run code style CI and commit to forks.
But yeah, it would run after merge.